mod-invoice-storage/schemas/invoice.json

```diff
@@ -139,14 +135,12 @@
       "type": "array",
       "items": {
         "description": "A human readable ID assigned to this purchase order",
-        "type": "string",
-        "pattern": "^[a-zA-Z0-9]{5,16}$"
+        "$ref": "../../common/schemas/uuid.json"
```

Comment:
This has to be the poNumber pattern

Reply:
Yeah. Actually, only one symbol should be updated by this PR and I have made a mistake :) Fixed